lib/engine-universal/src/artifact.rs

```diff
@@ -253,12 +257,12 @@ impl JITArtifact {
 
     /// Get the default extension when serializing this artifact
     pub fn get_default_extension(_triple: &Triple) -> &'static str {
-        // `.wjit` is the default extension for all the triples
-        "wjit"
+        // `.wuniversal` is the default extension for all the triples
+        "wuniversal"
```

Comment:
This part is important.

Reply:
I don't like the extension. I think a better name is `wasmu` or `wuni` (I incline more for `wasmu`)

Reply:
Go for `.wasmu` :-).